staging/src/k8s.io/sample-apiserver/artifacts/example/deployment.yaml

```diff
@@ -22,7 +23,7 @@ spec:
         # docker pull gcr.io/kubernetes-e2e-test-images/sample-apiserver:1.17
         # docker tag gcr.io/kubernetes-e2e-test-images/sample-apiserver:1.17 kube-sample-apiserver:latest
         image: kube-sample-apiserver:latest
-        imagePullPolicy: Never
+        imagePullPolicy: IfNotPresent
```

Comment:
given `kube-sample-apiserver:latest` is not expected to be able to be pulled from public registries (it depends on being present and tagged according to the instructions above), shouldn't we leave this as "Never"?

Reply:
Yep, changed to `Never`.